sigh/lib/assets/resign.sh

```diff
@@ -212,7 +212,7 @@ done
 KEYCHAIN_FLAG=
 if [ -n "$KEYCHAIN_PATH" ]
 then
-    KEYCHAIN_FLAG="--keychain '$KEYCHAIN_PATH'"
+    KEYCHAIN_FLAG="--keychain $KEYCHAIN_PATH"
```

Comment:
@ohwutup Yes, because in `sigh/lib/sigh/resign.rb` (line 38) the variable is being shell-escaped (as suggested [in this thread](https://github.com/fastlane/fastlane/pull/5057))